In [1]:
import pandas as pd

In [2]:
min_df = pd.read_csv('Datasets/Education Salaries/MinimumWage.csv')

In [3]:
min_df

Year  FederalMinimumWage  MeanAnnualInflation  \
0   1930                0.00                -0.02   
1   1931                0.00                -0.09   
2   1932                0.00                -0.10   
3   1933                0.00                -0.05   
4   1934                0.00                 0.03   
..   ...                 ...                  ...   
86  2016                7.25                 0.01   
87  2017                7.25                 0.02   
88  2018                7.25                 0.02   
89  2019                7.25                 0.02   
90  2020                7.25                 0.01   

    MinWageIndexedLastRaiseYear  UnemploymentRateDecember  GDP_AnnualGrowth  
0                           NaN                      0.09             -0.09  
1                           NaN                      0.16             -0.06  
2                           NaN                      0.24             -0.13  
3                           NaN                      0.25             -0.01  
4                           NaN                      0.22              0.11  
..                          ...                       ...               ...  
86                         6.46                      0.05              0.02  
87                         6.33                      0.04              0.02  
88                         6.17                      0.04              0.03  
89                         6.06                      0.04              0.02  
90                         5.99                      0.07             -0.03  

[91 rows x 6 columns]

In [4]:
min_df.mean()

Year                           1975.000000
FederalMinimumWage                2.857692
MeanAnnualInflation               0.031429
MinWageIndexedLastRaiseYear       2.871566
UnemploymentRateDecember          0.071209
GDP_AnnualGrowth                  0.033187
dtype: float64

In [5]:
min_df.sum()

Year                           179725.00
FederalMinimumWage                260.05
MeanAnnualInflation                 2.86
MinWageIndexedLastRaiseYear       238.34
UnemploymentRateDecember            6.48
GDP_AnnualGrowth                    3.02
dtype: float64

In [7]:
min_df['FederalMinimumWage'].value_counts()

7.25    12
5.15    10
3.35     9
0.00     8
0.75     6
0.30     6
1.60     6
1.00     5
4.25     5
0.40     5
1.25     4
2.30     2
1.15     2
0.25     1
2.65     1
2.10     1
3.10     1
2.00     1
3.80     1
1.40     1
4.75     1
5.85     1
6.55     1
2.90     1
Name: FederalMinimumWage, dtype: int64

Some quick item's of note, but not vital importance.

* Our first established minimum wage, in 1938, was 25 cents.
* over the course of this dataset (90  years) our average minimum wage is \\$2.85, amd \\$2.87 indexed to inflation.
* GDP has increased 302\% and an average of 3.31\% per year. 
* Inflation has increased 296\%, and an average of 3.14\% per year.
* 1975, the center of this dataset, had a minimum wage of 2.10.
* Unemployment rate is 7\% average, 1982(11\%) and 2009(10\%) being the only 2 in the last 50 years to break 10%
* 1997 got our minimum wage to \\$5.15, where it held for 10 years, it's only surpased by \\$7.25 the latest established minimum wage (for this dataset) for the last 12 years.

What is important to note, is the subsection that lines up with my other datasets. I'll be separating out the last 20 years of data for further analysis

In [8]:
min_df_cut = min_df.tail(20)

In [9]:
min_df_cut.to_csv('last_20_years_min_df.csv')